src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -571,6 +571,12 @@ public virtual void Start()
             if (_initialized == false)
                 throw new InvalidOperationException($"Index '{Name}' was not initialized.");
 
+            if (DocumentDatabase.IndexStore.IsDisposed.IsRaised())
```

Comment:
What happens if the database disposed _after_ this line?

Reply:
Then it should be OK, since this index was added to the `CollectionOfIndexes`.